### DISCUSSION TITLE
code examples contain _ underscores placed randomly, subsequent build doesn't have them

### DISCUSSION THREAD

**Discussion by sebastiangug** [storybookjs/storybook#24912] [Help]
I'm using fixed versions of storybook 7.x  -- this is a project that has been upgraded from SB6.x but everything worked fine, until one day someone noticed the code examples were full of these underscores: 

![image](https://github.com/storybookjs/storybook/assets/27768313/9df92b54-4629-489c-8ee6-29f92ec97002)

A subsequent build with no real changes to anything but a random component seems to have fixed the issue for all components. 

The problem is I don't understand why this happened and how to prevent it from happening again randomly. 

Anyone else experienced this or have any ideas on why this might have happened? where to look?

**Comment by ShaunEvening** [comment]
Hey @sebastiangug  It's odd that this fixed itself 🤔 

My best guess is that your components got minified and had their display names removed.

Can you tell me what version of Storybook that you're using and what view layer you're using?

**Comment by sebastiangug** [comment]
@Integrayshaun  thanks for getting back to me, that makes sense. 

The storybook version we've had this on is:  7.5.1

The view layer is react via vite defined as: 

  framework: { name: '@storybook/react-vite', options: {} },

**Reply by ShaunEvening** [reply]
Interesting! I'm honestly not sure why that happened. If it happens again can you send us a reproduction so that we can find out why that might have happened?

**Reply by sebastiangug** [reply]
Sure, keeping an eye out for it, quite a bit of a weird one. I have since upgraded everything that was 7.5.1 to 7.5.3 though but in the future if it ever happens to anyone at least this thread will surface, we can close the discussion for now though
